docs/articles/documentation/test-api/actions/select-text.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: docs
+title: Select Text
+permalink: /documentation/test-api/actions/select-text.html
+---
+# Select Text
+
+There are different kinds of the Select Text action for selecting regular text, `<textarea>` and `contentEditable` content.
+
+* [Select Regular Text](#select-regular-text)
+* [Select \<textarea\> Content](#select-textarea-content)
+* [Perform Selection within Editable Content](#perform-selection-within-editable-content)
+
+## Select Regular Text
```

Comment:
I don't know what does mean `regular text`. I think we should specify elements for this action: `input, textarea, contenteditable elements`